evals/gdm_capabilities/in_house_ctf/data/ssh/compose.yaml

```diff
@@ -1,16 +1,16 @@
 services:
   ssh:
     image: marshw/ssh:latest
-    container_name: ssh_env
+    hostname: ssh_env
     networks:
       - ssh_network
-    ports:
-      - "23:22"
+    expose:
+      - 22
```

Comment:
Note: This used to map container port 22 to host port 23 (presumably because another challenge already used 22 on the host).
I've updated the address in challenges.yaml to now reference port 22.

Reply:
We elsewhere changed port 22 to port 33 (https://github.com/UKGovernmentBEIS/inspect_ai/commit/5fc903f2ec4d69147ba28c71c74b3edd27d1dd9e). Should we change this back as well?

We also changed the postgress port in db_3 from 5432 to 5555 -- should we change that back as well? (it needs updating both in the compose file and in the challenge JSON)